### PR TITLE
Powerbi

### PR DESCRIPTION
**Pull request by estebanrodriguez28**
Added dashboard UI to display visuals

### CODE REVIEW

**Review by KentMorris509 — APPROVED** [review]
Code looks great Esteban, way to figure out how to embed the Power BI report and create the sidebar for the dashboard. Next steps would be to comment your code and then get working on the testing of routes and creating some of the other routes for the sidebar buttons and mocking our database of integration testing purposes

**Comment by SDjon** [comment]
[Implement graph to view, per month, how many unique students ordered breakfast](https://trello.com/c/6PyOsARh/19-implement-graph-to-view-per-month-how-many-unique-students-ordered-breakfast)

**Comment by KentMorris509** [comment]
[Implement stat to view how many students ordered breakfast](https://trello.com/c/s5JVvfDg/16-implement-stat-to-view-how-many-students-ordered-breakfast)

File: templates/index.html [hunk 1]
```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Dashboard</title>
+    <link rel="stylesheet" href="{{ url_for('static', filename='style.css') }}">
+    <link href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@400;500&display=swap" rel="stylesheet">
+
```
Comment (SDjon) [inline templates/index.html:10]: To keep the header consistent with other html parts there should be no whitespace here

File: templates/index.html [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+                    <span class="role">Admin</span>
+                </div>
+            </header>
+
+            <div id="reportContainer">
+                <iframe title="totalRivera" width="1140" height="541.25"
+                    src="https://app.powerbi.com/reportEmbed?reportId=8857d370-78fa-4a90-9ccd-e94e859f8ef3&autoAuth=true&ctid=fe4e1583-fb6f-48dd-8524-d06ad7501045"
+                    frameborder="0" allowFullScreen="true"></iframe>
+            </div>
+
```
Comment (SDjon) [inline templates/index.html:51]: Remove this whitespace at the bottom to keep it consistent

**Pull request merged**